cmd/lncli/walletrpc_active.go

```diff
@@ -265,15 +265,18 @@ func bumpFee(ctx *cli.Context) error {
 }
 
 var bumpCloseFeeCommand = cli.Command{
-	Name:      "bumpclosefee",
-	Usage:     "Bumps the fee of a channel closing transaction.",
+	Name:      "bumpforceclosefee",
+	Usage:     "Bumps the fee of a channel force closing transaction.",
 	ArgsUsage: "channel_point",
 	Description: `
-	This command allows the fee of a channel closing transaction to be
-	increased by using the child-pays-for-parent mechanism. It will instruct
-	the sweeper to sweep the anchor outputs of transactions in the set
-	of valid commitments for the specified channel at the requested fee
-	rate or confirmation target.
+	This command works only for unilateral closes of anchor channels. It
+	allows the fee of a channel force closing transaction to be increased by
+	using the child-pays-for-parent mechanism. It will instruct the sweeper
+	to sweep the anchor outputs of the closing transaction at the requested
+	fee rate or confirmation target. The specified fee rate will be the
+	effective fee rate taking the parent fee into account.
+	Depending on the sweeper configuration (batchwindowduration) the sweeptx
+	will not be published immediately.
```

Comment:
We might as well mention here that an empty response of `{}` indicates success and that only the log (or `lncli pendingsweeps`) shows more info.

Reply:
I added a status to `BumpFeeResponse`, but there was already a logline when the corresponding anchor was found but now I include the status as well:

```
Bumping fee of ed0e3a761f38007bd58358bfeae53e929654db8db1fc76eeae0af8cfb84cd145:0: Successfully registered rbf-tx with sweeper
Bumping fee of 7ae1014949ee85732e65a28e0b2135ce18b939ef86f2b44e5e45749b933b0e9e:0: Successfully registered rbf-tx with sweeper
```

wdyt ?

Reply:
Looks great, thanks!